roachtest: lib/pq failed

[(roachtest).lib/pq failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1653424&tab=buildLog) on [master@277e28b2ea78929563e85cb1c9efc573f37408a4](https://github.com/cockroachdb/cockroach/commits/277e28b2ea78929563e85cb1c9efc573f37408a4):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191219-1653424/lib/pq/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,libpq.go:99,test_runner.go:700: 
		Tests run on Cockroach v20.1.0-alpha.20191118-827-g277e28b
		Tests run against lib/pq v1.3.0
		168 Total Tests Run
		124 tests passed
		44 tests failed
		0 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestCopyInStmtAffectedRows - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blacklist (libPQBlacklist20_1) is available in the artifacts' lib/pq log
```

<details><summary>Repro</summary><p>


Artifacts: [/lib/pq](https://teamcity.cockroachdb.com/viewLog.html?buildId=1653424&tab=artifacts#/lib/pq)
```
make stressrace TESTS=lib/pq PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).lib/pq failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1653460&tab=buildLog) on [release-19.2@0ae55ddaac1f0d18f4ecaa1c2f48c1aafe4c5c2d](https://github.com/cockroachdb/cockroach/commits/0ae55ddaac1f0d18f4ecaa1c2f48c1aafe4c5c2d):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191219-1653460/lib/pq/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,libpq.go:99,test_runner.go:700: 
		Tests run on Cockroach v19.2.2-14-g4ada24e
		Tests run against lib/pq v1.3.0
		168 Total Tests Run
		123 tests passed
		45 tests failed
		0 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestCopyInStmtAffectedRows - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blacklist (libPQBlacklist19_2) is available in the artifacts' lib/pq log
```

<details><summary>Repro</summary><p>


Artifacts: [/lib/pq](https://teamcity.cockroachdb.com/viewLog.html?buildId=1653460&tab=artifacts#/lib/pq)
```
make stressrace TESTS=lib/pq PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).lib/pq failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1656773&tab=buildLog) on [master@bf8d6db2ef9b66d301996cbb8ebbb15b7c978d1a](https://github.com/cockroachdb/cockroach/commits/bf8d6db2ef9b66d301996cbb8ebbb15b7c978d1a):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191220-1656773/lib/pq/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,libpq.go:99,test_runner.go:700: 
		Tests run on Cockroach v20.1.0-alpha.20191118-858-gbf8d6db
		Tests run against lib/pq v1.3.0
		168 Total Tests Run
		124 tests passed
		44 tests failed
		0 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestCopyInStmtAffectedRows - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blacklist (libPQBlacklist20_1) is available in the artifacts' lib/pq log
```

<details><summary>Repro</summary><p>


Artifacts: [/lib/pq](https://teamcity.cockroachdb.com/viewLog.html?buildId=1656773&tab=artifacts#/lib/pq)
```
make stressrace TESTS=lib/pq PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).lib/pq failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1656791&tab=buildLog) on [release-19.2@a84266e50207e2e9a27828991ec9fc2f62d74a88](https://github.com/cockroachdb/cockroach/commits/a84266e50207e2e9a27828991ec9fc2f62d74a88):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191220-1656791/lib/pq/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,libpq.go:99,test_runner.go:700: 
		Tests run on Cockroach v19.2.2-21-ga84266e
		Tests run against lib/pq v1.3.0
		168 Total Tests Run
		123 tests passed
		45 tests failed
		0 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestCopyInStmtAffectedRows - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blacklist (libPQBlacklist19_2) is available in the artifacts' lib/pq log
```

<details><summary>Repro</summary><p>


Artifacts: [/lib/pq](https://teamcity.cockroachdb.com/viewLog.html?buildId=1656791&tab=artifacts#/lib/pq)
```
make stressrace TESTS=lib/pq PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).lib/pq failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1658821&tab=buildLog) on [release-19.2@8da55b5585cc2e3dd3b87b9d01d2000c03ea9953](https://github.com/cockroachdb/cockroach/commits/8da55b5585cc2e3dd3b87b9d01d2000c03ea9953):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191221-1658821/lib/pq/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,libpq.go:99,test_runner.go:700: 
		Tests run on Cockroach v19.2.2-23-g8da55b5
		Tests run against lib/pq v1.3.0
		168 Total Tests Run
		123 tests passed
		45 tests failed
		0 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestCopyInStmtAffectedRows - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blacklist (libPQBlacklist19_2) is available in the artifacts' lib/pq log
```

<details><summary>Repro</summary><p>


Artifacts: [/lib/pq](https://teamcity.cockroachdb.com/viewLog.html?buildId=1658821&tab=artifacts#/lib/pq)
```
make stressrace TESTS=lib/pq PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).lib/pq failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1658785&tab=buildLog) on [master@dd9b1c1f40dda59ee9d446416106d311ae5ce1e6](https://github.com/cockroachdb/cockroach/commits/dd9b1c1f40dda59ee9d446416106d311ae5ce1e6):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191221-1658785/lib/pq/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,libpq.go:99,test_runner.go:700: 
		Tests run on Cockroach v20.1.0-alpha.20191118-884-gdd9b1c1
		Tests run against lib/pq v1.3.0
		168 Total Tests Run
		124 tests passed
		44 tests failed
		0 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestCopyInStmtAffectedRows - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blacklist (libPQBlacklist20_1) is available in the artifacts' lib/pq log
```

<details><summary>Repro</summary><p>


Artifacts: [/lib/pq](https://teamcity.cockroachdb.com/viewLog.html?buildId=1658785&tab=artifacts#/lib/pq)
```
make stressrace TESTS=lib/pq PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).lib/pq failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1660313&tab=buildLog) on [master@319080c701cb4e13c347e86c4afb2f1c2af78def](https://github.com/cockroachdb/cockroach/commits/319080c701cb4e13c347e86c4afb2f1c2af78def):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191222-1660313/lib/pq/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,libpq.go:99,test_runner.go:700: 
		Tests run on Cockroach v20.1.0-alpha.20191118-888-g319080c
		Tests run against lib/pq v1.3.0
		168 Total Tests Run
		124 tests passed
		44 tests failed
		0 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestCopyInStmtAffectedRows - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blacklist (libPQBlacklist20_1) is available in the artifacts' lib/pq log
```

<details><summary>Repro</summary><p>


Artifacts: [/lib/pq](https://teamcity.cockroachdb.com/viewLog.html?buildId=1660313&tab=artifacts#/lib/pq)
```
make stressrace TESTS=lib/pq PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).lib/pq failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1661413&tab=buildLog) on [release-19.2@d23a3df4b9eddd07f02b67b6beb16aee8fd8e65b](https://github.com/cockroachdb/cockroach/commits/d23a3df4b9eddd07f02b67b6beb16aee8fd8e65b):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191224-1661413/lib/pq/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,libpq.go:99,test_runner.go:716: 
		Tests run on Cockroach v19.2.2-25-gd23a3df
		Tests run against lib/pq v1.3.0
		168 Total Tests Run
		123 tests passed
		45 tests failed
		0 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestCopyInStmtAffectedRows - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blacklist (libPQBlacklist19_2) is available in the artifacts' lib/pq log
```

<details><summary>Repro</summary><p>


Artifacts: [/lib/pq](https://teamcity.cockroachdb.com/viewLog.html?buildId=1661413&tab=artifacts#/lib/pq)
```
make stressrace TESTS=lib/pq PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).lib/pq failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1661395&tab=buildLog) on [master@a3f81bcbef596f8b8134641526a87f27c4ad8082](https://github.com/cockroachdb/cockroach/commits/a3f81bcbef596f8b8134641526a87f27c4ad8082):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191224-1661395/lib/pq/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,libpq.go:99,test_runner.go:716: 
		Tests run on Cockroach v20.1.0-alpha.20191118-898-ga3f81bc
		Tests run against lib/pq v1.3.0
		168 Total Tests Run
		124 tests passed
		44 tests failed
		0 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestCopyInStmtAffectedRows - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blacklist (libPQBlacklist20_1) is available in the artifacts' lib/pq log
```

<details><summary>Repro</summary><p>


Artifacts: [/lib/pq](https://teamcity.cockroachdb.com/viewLog.html?buildId=1661395&tab=artifacts#/lib/pq)
```
make stressrace TESTS=lib/pq PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).lib/pq failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1662271&tab=buildLog) on [master@c172b37ec3316ec982e1ee92d4414a76823680e6](https://github.com/cockroachdb/cockroach/commits/c172b37ec3316ec982e1ee92d4414a76823680e6):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191225-1662271/lib/pq/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,libpq.go:99,test_runner.go:716: 
		Tests run on Cockroach v20.1.0-alpha.20191118-900-gc172b37
		Tests run against lib/pq v1.3.0
		168 Total Tests Run
		124 tests passed
		44 tests failed
		0 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestCopyInStmtAffectedRows - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blacklist (libPQBlacklist20_1) is available in the artifacts' lib/pq log
```

<details><summary>Repro</summary><p>


Artifacts: [/lib/pq](https://teamcity.cockroachdb.com/viewLog.html?buildId=1662271&tab=artifacts#/lib/pq)
```
make stressrace TESTS=lib/pq PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).lib/pq failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1662289&tab=buildLog) on [release-19.2@b4efe661f4bff33554fe41bd4a1a5f245a6fde2b](https://github.com/cockroachdb/cockroach/commits/b4efe661f4bff33554fe41bd4a1a5f245a6fde2b):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191225-1662289/lib/pq/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,libpq.go:99,test_runner.go:716: 
		Tests run on Cockroach v19.2.2-29-gb4efe66
		Tests run against lib/pq v1.3.0
		168 Total Tests Run
		123 tests passed
		45 tests failed
		0 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestCopyInStmtAffectedRows - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blacklist (libPQBlacklist19_2) is available in the artifacts' lib/pq log
```

<details><summary>Repro</summary><p>


Artifacts: [/lib/pq](https://teamcity.cockroachdb.com/viewLog.html?buildId=1662289&tab=artifacts#/lib/pq)
```
make stressrace TESTS=lib/pq PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).lib/pq failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1663134&tab=buildLog) on [master@2ddb82b2a8fff0c4e42169cf4e610d33a0fbd2d3](https://github.com/cockroachdb/cockroach/commits/2ddb82b2a8fff0c4e42169cf4e610d33a0fbd2d3):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191226-1663134/lib/pq/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,libpq.go:99,test_runner.go:716: 
		Tests run on Cockroach v20.1.0-alpha.20191118-907-g2ddb82b
		Tests run against lib/pq v1.3.0
		168 Total Tests Run
		124 tests passed
		44 tests failed
		0 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestCopyInStmtAffectedRows - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blacklist (libPQBlacklist20_1) is available in the artifacts' lib/pq log
```

<details><summary>Repro</summary><p>


Artifacts: [/lib/pq](https://teamcity.cockroachdb.com/viewLog.html?buildId=1663134&tab=artifacts#/lib/pq)
```
make stressrace TESTS=lib/pq PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).lib/pq failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1664318&tab=buildLog) on [master@d54e001248f233734ff926daef5470487c5616b0](https://github.com/cockroachdb/cockroach/commits/d54e001248f233734ff926daef5470487c5616b0):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191227-1664318/lib/pq/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,libpq.go:99,test_runner.go:716: 
		Tests run on Cockroach v20.1.0-alpha.20191118-925-gd54e001
		Tests run against lib/pq v1.3.0
		168 Total Tests Run
		124 tests passed
		44 tests failed
		0 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestCopyInStmtAffectedRows - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blacklist (libPQBlacklist20_1) is available in the artifacts' lib/pq log
```

<details><summary>Repro</summary><p>


Artifacts: [/lib/pq](https://teamcity.cockroachdb.com/viewLog.html?buildId=1664318&tab=artifacts#/lib/pq)
```
make stressrace TESTS=lib/pq PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).lib/pq failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1664336&tab=buildLog) on [release-19.2@599c988773fa6fb4544369a50fe8d06b90c64171](https://github.com/cockroachdb/cockroach/commits/599c988773fa6fb4544369a50fe8d06b90c64171):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191227-1664336/lib/pq/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,libpq.go:99,test_runner.go:716: 
		Tests run on Cockroach v19.2.2-31-g599c988
		Tests run against lib/pq v1.3.0
		168 Total Tests Run
		123 tests passed
		45 tests failed
		0 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestCopyInStmtAffectedRows - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blacklist (libPQBlacklist19_2) is available in the artifacts' lib/pq log
```

<details><summary>Repro</summary><p>


Artifacts: [/lib/pq](https://teamcity.cockroachdb.com/viewLog.html?buildId=1664336&tab=artifacts#/lib/pq)
```
make stressrace TESTS=lib/pq PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).lib/pq failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1665456&tab=buildLog) on [master@8f946b8bb62629002c5e958373b81ca9e1920dd6](https://github.com/cockroachdb/cockroach/commits/8f946b8bb62629002c5e958373b81ca9e1920dd6):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191228-1665456/lib/pq/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,libpq.go:99,test_runner.go:716: 
		Tests run on Cockroach v20.1.0-alpha.20191118-933-g8f946b8
		Tests run against lib/pq v1.3.0
		168 Total Tests Run
		124 tests passed
		44 tests failed
		0 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestCopyInStmtAffectedRows - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blacklist (libPQBlacklist20_1) is available in the artifacts' lib/pq log
```

<details><summary>Repro</summary><p>


Artifacts: [/lib/pq](https://teamcity.cockroachdb.com/viewLog.html?buildId=1665456&tab=artifacts#/lib/pq)
```
make stressrace TESTS=lib/pq PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).lib/pq failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1665474&tab=buildLog) on [release-19.2@8ca08b3ddbfa07eb8fc19a1035cf82539f05c7a3](https://github.com/cockroachdb/cockroach/commits/8ca08b3ddbfa07eb8fc19a1035cf82539f05c7a3):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191228-1665474/lib/pq/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,libpq.go:99,test_runner.go:716: 
		Tests run on Cockroach v19.2.2-35-g8ca08b3
		Tests run against lib/pq v1.3.0
		168 Total Tests Run
		123 tests passed
		45 tests failed
		0 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestCopyInStmtAffectedRows - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blacklist (libPQBlacklist19_2) is available in the artifacts' lib/pq log
```

<details><summary>Repro</summary><p>


Artifacts: [/lib/pq](https://teamcity.cockroachdb.com/viewLog.html?buildId=1665474&tab=artifacts#/lib/pq)
```
make stressrace TESTS=lib/pq PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).lib/pq failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1666388&tab=buildLog) on [master@edd46b1405b7928760b65ec8aad59fd22a8adb6b](https://github.com/cockroachdb/cockroach/commits/edd46b1405b7928760b65ec8aad59fd22a8adb6b):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191229-1666388/lib/pq/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,libpq.go:99,test_runner.go:716: 
		Tests run on Cockroach v20.1.0-alpha.20191118-938-gedd46b1
		Tests run against lib/pq v1.3.0
		168 Total Tests Run
		124 tests passed
		44 tests failed
		0 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestCopyInStmtAffectedRows - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blacklist (libPQBlacklist20_1) is available in the artifacts' lib/pq log
```

<details><summary>Repro</summary><p>


Artifacts: [/lib/pq](https://teamcity.cockroachdb.com/viewLog.html?buildId=1666388&tab=artifacts#/lib/pq)
```
make stressrace TESTS=lib/pq PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).lib/pq failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1668189&tab=buildLog) on [master@f1f15250871d124178f40bf340ef48719b50e7e5](https://github.com/cockroachdb/cockroach/commits/f1f15250871d124178f40bf340ef48719b50e7e5):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191231-1668189/lib/pq/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,libpq.go:99,test_runner.go:716: 
		Tests run on Cockroach v20.1.0-alpha.20191118-942-gf1f1525
		Tests run against lib/pq v1.3.0
		168 Total Tests Run
		124 tests passed
		44 tests failed
		0 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestCopyInStmtAffectedRows - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blacklist (libPQBlacklist20_1) is available in the artifacts' lib/pq log
```

<details><summary>Repro</summary><p>


Artifacts: [/lib/pq](https://teamcity.cockroachdb.com/viewLog.html?buildId=1668189&tab=artifacts#/lib/pq)
```
make stressrace TESTS=lib/pq PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).lib/pq failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1668207&tab=buildLog) on [release-19.2@3960c23e7b8fef3042d525f74768c5ae85a1cf22](https://github.com/cockroachdb/cockroach/commits/3960c23e7b8fef3042d525f74768c5ae85a1cf22):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191231-1668207/lib/pq/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,libpq.go:99,test_runner.go:716: 
		Tests run on Cockroach v19.2.2-37-g3960c23
		Tests run against lib/pq v1.3.0
		168 Total Tests Run
		123 tests passed
		45 tests failed
		0 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestCopyInStmtAffectedRows - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blacklist (libPQBlacklist19_2) is available in the artifacts' lib/pq log
```

<details><summary>Repro</summary><p>


Artifacts: [/lib/pq](https://teamcity.cockroachdb.com/viewLog.html?buildId=1668207&tab=artifacts#/lib/pq)
```
make stressrace TESTS=lib/pq PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).lib/pq failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1669512&tab=buildLog) on [master@0d1ae9cbc49518180e68aa0bfbf982d73854ddae](https://github.com/cockroachdb/cockroach/commits/0d1ae9cbc49518180e68aa0bfbf982d73854ddae):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200101-1669512/lib/pq/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,libpq.go:99,test_runner.go:716: 
		Tests run on Cockroach v20.1.0-alpha.20191118-945-g0d1ae9c
		Tests run against lib/pq v1.3.0
		168 Total Tests Run
		124 tests passed
		44 tests failed
		0 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestCopyInStmtAffectedRows - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blacklist (libPQBlacklist20_1) is available in the artifacts' lib/pq log
```

<details><summary>Repro</summary><p>


Artifacts: [/lib/pq](https://teamcity.cockroachdb.com/viewLog.html?buildId=1669512&tab=artifacts#/lib/pq)
```
make stressrace TESTS=lib/pq PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).lib/pq failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1670273&tab=buildLog) on [master@a99740bb34c03ab3af7558129edce9c17c88d4f9](https://github.com/cockroachdb/cockroach/commits/a99740bb34c03ab3af7558129edce9c17c88d4f9):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200102-1670273/lib/pq/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,libpq.go:99,test_runner.go:716: 
		Tests run on Cockroach v20.1.0-alpha.20191118-947-ga99740b
		Tests run against lib/pq v1.3.0
		168 Total Tests Run
		124 tests passed
		44 tests failed
		0 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestCopyInStmtAffectedRows - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blacklist (libPQBlacklist20_1) is available in the artifacts' lib/pq log
```

<details><summary>Repro</summary><p>


Artifacts: [/lib/pq](https://teamcity.cockroachdb.com/viewLog.html?buildId=1670273&tab=artifacts#/lib/pq)
```
make stressrace TESTS=lib/pq PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).lib/pq failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1671769&tab=buildLog) on [release-19.2@9aaf5fb6f7108858d200dd3d199deb11871a09b4](https://github.com/cockroachdb/cockroach/commits/9aaf5fb6f7108858d200dd3d199deb11871a09b4):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200103-1671769/lib/pq/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,libpq.go:99,test_runner.go:716: 
		Tests run on Cockroach v19.2.2-44-g9aaf5fb
		Tests run against lib/pq v1.3.0
		168 Total Tests Run
		123 tests passed
		45 tests failed
		0 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestCopyInStmtAffectedRows - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blacklist (libPQBlacklist19_2) is available in the artifacts' lib/pq log
```

<details><summary>Repro</summary><p>


Artifacts: [/lib/pq](https://teamcity.cockroachdb.com/viewLog.html?buildId=1671769&tab=artifacts#/lib/pq)
```
make stressrace TESTS=lib/pq PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).lib/pq failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1671733&tab=buildLog) on [master@8facb469d66426a3f372a1aed4ea6f34e287b501](https://github.com/cockroachdb/cockroach/commits/8facb469d66426a3f372a1aed4ea6f34e287b501):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200103-1671733/lib/pq/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,libpq.go:99,test_runner.go:716: 
		Tests run on Cockroach v20.1.0-alpha.20191118-981-g8facb46
		Tests run against lib/pq v1.3.0
		168 Total Tests Run
		124 tests passed
		44 tests failed
		0 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestCopyInStmtAffectedRows - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blacklist (libPQBlacklist20_1) is available in the artifacts' lib/pq log
```

<details><summary>Repro</summary><p>


Artifacts: [/lib/pq](https://teamcity.cockroachdb.com/viewLog.html?buildId=1671733&tab=artifacts#/lib/pq)
```
make stressrace TESTS=lib/pq PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).lib/pq failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1673622&tab=buildLog) on [master@7906cbe3d9c615430f6e892a42e6bccdad2aff00](https://github.com/cockroachdb/cockroach/commits/7906cbe3d9c615430f6e892a42e6bccdad2aff00):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200104-1673622/lib/pq/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,libpq.go:99,test_runner.go:716: 
		Tests run on Cockroach v20.1.0-alpha.20191118-995-g7906cbe
		Tests run against lib/pq v1.3.0
		168 Total Tests Run
		124 tests passed
		44 tests failed
		0 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestCopyInStmtAffectedRows - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blacklist (libPQBlacklist20_1) is available in the artifacts' lib/pq log
```

<details><summary>Repro</summary><p>


Artifacts: [/lib/pq](https://teamcity.cockroachdb.com/viewLog.html?buildId=1673622&tab=artifacts#/lib/pq)
```
make stressrace TESTS=lib/pq PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).lib/pq failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1673640&tab=buildLog) on [release-19.2@6cc3423103f85d7f82f01791a24d99bef21b2546](https://github.com/cockroachdb/cockroach/commits/6cc3423103f85d7f82f01791a24d99bef21b2546):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200104-1673640/lib/pq/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,libpq.go:99,test_runner.go:716: 
		Tests run on Cockroach v19.2.2-47-g6cc3423
		Tests run against lib/pq v1.3.0
		168 Total Tests Run
		123 tests passed
		45 tests failed
		0 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestCopyInStmtAffectedRows - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blacklist (libPQBlacklist19_2) is available in the artifacts' lib/pq log
```

<details><summary>Repro</summary><p>


Artifacts: [/lib/pq](https://teamcity.cockroachdb.com/viewLog.html?buildId=1673640&tab=artifacts#/lib/pq)
```
make stressrace TESTS=lib/pq PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).lib/pq failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1674751&tab=buildLog) on [master@f180501b7c2ccb946d7f7c8944d5509b87cb0394](https://github.com/cockroachdb/cockroach/commits/f180501b7c2ccb946d7f7c8944d5509b87cb0394):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200106-1674751/lib/pq/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,libpq.go:99,test_runner.go:716: 
		Tests run on Cockroach v20.1.0-alpha.20191118-1001-gf180501
		Tests run against lib/pq v1.3.0
		168 Total Tests Run
		124 tests passed
		44 tests failed
		0 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestCopyInStmtAffectedRows - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blacklist (libPQBlacklist20_1) is available in the artifacts' lib/pq log
```

<details><summary>Repro</summary><p>


Artifacts: [/lib/pq](https://teamcity.cockroachdb.com/viewLog.html?buildId=1674751&tab=artifacts#/lib/pq)
```
make stressrace TESTS=lib/pq PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).lib/pq failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1676232&tab=buildLog) on [43008@b555d16b4455f7122f4211082512820742f8116f](https://github.com/cockroachdb/cockroach/commits/b555d16b4455f7122f4211082512820742f8116f):

```
The test failed on branch=43008, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200106-1676232/lib/pq/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,libpq.go:99,test_runner.go:716: 
		Tests run on Cockroach v20.1.0-alpha.20191118-1011-gb555d16
		Tests run against lib/pq v1.3.0
		168 Total Tests Run
		124 tests passed
		44 tests failed
		0 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestCopyInStmtAffectedRows - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blacklist (libPQBlacklist20_1) is available in the artifacts' lib/pq log
```

<details><summary>Repro</summary><p>


Artifacts: [/lib/pq](https://teamcity.cockroachdb.com/viewLog.html?buildId=1676232&tab=artifacts#/lib/pq)
```
make stressrace TESTS=lib/pq PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).lib/pq failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1676715&tab=buildLog) on [master@52261c39c65aecf960e3c8d54089a66d378af610](https://github.com/cockroachdb/cockroach/commits/52261c39c65aecf960e3c8d54089a66d378af610):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200107-1676715/lib/pq/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,libpq.go:99,test_runner.go:716: 
		Tests run on Cockroach v20.1.0-alpha.20191118-1030-gd09fd7f
		Tests run against lib/pq v1.3.0
		168 Total Tests Run
		124 tests passed
		44 tests failed
		0 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestCopyInStmtAffectedRows - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blacklist (libPQBlacklist20_1) is available in the artifacts' lib/pq log
```

<details><summary>Repro</summary><p>


Artifacts: [/lib/pq](https://teamcity.cockroachdb.com/viewLog.html?buildId=1676715&tab=artifacts#/lib/pq)
```
make stressrace TESTS=lib/pq PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).lib/pq failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1676733&tab=buildLog) on [release-19.2@f3bcbd305602c89d52cb7ea4cb114f362c5aee23](https://github.com/cockroachdb/cockroach/commits/f3bcbd305602c89d52cb7ea4cb114f362c5aee23):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200107-1676733/lib/pq/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,libpq.go:99,test_runner.go:716: 
		Tests run on Cockroach v19.2.2-49-gf3bcbd3
		Tests run against lib/pq v1.3.0
		168 Total Tests Run
		123 tests passed
		45 tests failed
		0 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestCopyInStmtAffectedRows - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blacklist (libPQBlacklist19_2) is available in the artifacts' lib/pq log
```

<details><summary>Repro</summary><p>


Artifacts: [/lib/pq](https://teamcity.cockroachdb.com/viewLog.html?buildId=1676733&tab=artifacts#/lib/pq)
```
make stressrace TESTS=lib/pq PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).lib/pq failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1688037&tab=buildLog) on [release-19.2@7f2413b36fe1ea31a08c3ddd0d1e4b1cd006db37](https://github.com/cockroachdb/cockroach/commits/7f2413b36fe1ea31a08c3ddd0d1e4b1cd006db37):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200114-1688037/lib/pq/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,libpq.go:99,test_runner.go:716: 
		Tests run on Cockroach v19.2.2-72-g7f2413b
		Tests run against lib/pq v1.3.0
		168 Total Tests Run
		123 tests passed
		45 tests failed
		0 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestCopyInStmtAffectedRows - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blacklist (libPQBlacklist19_2) is available in the artifacts' lib/pq log
```

<details><summary>Repro</summary><p>


Artifacts: [/lib/pq](https://teamcity.cockroachdb.com/viewLog.html?buildId=1688037&tab=artifacts#/lib/pq)
```
make stressrace TESTS=lib/pq PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).lib/pq failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1688019&tab=buildLog) on [master@3d98dae853edbde083cc23b60ad6696958909849](https://github.com/cockroachdb/cockroach/commits/3d98dae853edbde083cc23b60ad6696958909849):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200114-1688019/lib/pq/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,libpq.go:99,test_runner.go:716: 
		Tests run on Cockroach v20.1.0-alpha.20191118-1206-g3d98dae
		Tests run against lib/pq v1.3.0
		168 Total Tests Run
		124 tests passed
		44 tests failed
		0 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestCopyInStmtAffectedRows - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blacklist (libPQBlacklist20_1) is available in the artifacts' lib/pq log
```

<details><summary>Repro</summary><p>


Artifacts: [/lib/pq](https://teamcity.cockroachdb.com/viewLog.html?buildId=1688019&tab=artifacts#/lib/pq)
```
make stressrace TESTS=lib/pq PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).lib/pq failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1690462&tab=buildLog) on [release-19.2@99a4ab5a34b310338098a6aafb6621920fd9d4cc](https://github.com/cockroachdb/cockroach/commits/99a4ab5a34b310338098a6aafb6621920fd9d4cc):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200115-1690462/lib/pq/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,libpq.go:99,test_runner.go:716: 
		Tests run on Cockroach v19.2.2-76-g99a4ab5
		Tests run against lib/pq v1.3.0
		168 Total Tests Run
		123 tests passed
		45 tests failed
		0 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestCopyInStmtAffectedRows - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blacklist (libPQBlacklist19_2) is available in the artifacts' lib/pq log
```

<details><summary>Repro</summary><p>


Artifacts: [/lib/pq](https://teamcity.cockroachdb.com/viewLog.html?buildId=1690462&tab=artifacts#/lib/pq)
```
make stressrace TESTS=lib/pq PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).lib/pq failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1690426&tab=buildLog) on [master@0046d62b00aa9e74489d1f44125948c9af729e2b](https://github.com/cockroachdb/cockroach/commits/0046d62b00aa9e74489d1f44125948c9af729e2b):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200115-1690426/lib/pq/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,libpq.go:99,test_runner.go:716: 
		Tests run on Cockroach v20.1.0-alpha.20191118-1249-g0046d62
		Tests run against lib/pq v1.3.0
		168 Total Tests Run
		124 tests passed
		44 tests failed
		0 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestCopyInStmtAffectedRows - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blacklist (libPQBlacklist20_1) is available in the artifacts' lib/pq log
```

<details><summary>Repro</summary><p>


Artifacts: [/lib/pq](https://teamcity.cockroachdb.com/viewLog.html?buildId=1690426&tab=artifacts#/lib/pq)
```
make stressrace TESTS=lib/pq PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).lib/pq failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1693632&tab=buildLog) on [release-19.2@297c398803afd80276db8dc78b3f7294822c87a8](https://github.com/cockroachdb/cockroach/commits/297c398803afd80276db8dc78b3f7294822c87a8):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200116-1693632/lib/pq/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,libpq.go:99,test_runner.go:716: 
		Tests run on Cockroach v19.2.2-83-g297c398
		Tests run against lib/pq v1.3.0
		168 Total Tests Run
		123 tests passed
		45 tests failed
		0 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestCopyInStmtAffectedRows - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blacklist (libPQBlacklist19_2) is available in the artifacts' lib/pq log
```

<details><summary>Repro</summary><p>


Artifacts: [/lib/pq](https://teamcity.cockroachdb.com/viewLog.html?buildId=1693632&tab=artifacts#/lib/pq)
```
make stressrace TESTS=lib/pq PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).lib/pq failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1693614&tab=buildLog) on [master@e2c2be6cbef07c7f2ee6341816f81f72a206c32a](https://github.com/cockroachdb/cockroach/commits/e2c2be6cbef07c7f2ee6341816f81f72a206c32a):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200116-1693614/lib/pq/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,libpq.go:99,test_runner.go:716: 
		Tests run on Cockroach v20.1.0-alpha.20191118-1277-g379fc8f
		Tests run against lib/pq v1.3.0
		168 Total Tests Run
		124 tests passed
		44 tests failed
		0 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestCopyInStmtAffectedRows - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blacklist (libPQBlacklist20_1) is available in the artifacts' lib/pq log
```

<details><summary>Repro</summary><p>


Artifacts: [/lib/pq](https://teamcity.cockroachdb.com/viewLog.html?buildId=1693614&tab=artifacts#/lib/pq)
```
make stressrace TESTS=lib/pq PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/cmd/roachtest/libpq_blacklist.go
+++ b/pkg/cmd/roachtest/libpq_blacklist.go
@@ -31,6 +31,7 @@ var libPQBlacklist20_1 = blacklist{
 	"pq.TestCopyInBinaryError":           "5807",
 	"pq.TestCopyInMultipleValues":        "5807",
 	"pq.TestCopyInRaiseStmtTrigger":      "5807",
+	"pq.TestCopyInStmtAffectedRows":      "5807",
 	"pq.TestCopyInTypes":                 "5807",
 	"pq.TestCopyInWrongType":             "5807",
 	"pq.TestCopyRespLoopConnectionError": "5807",
@@ -77,6 +78,7 @@ var libPQBlacklist19_2 = blacklist{
 	"pq.TestCopyInBinaryError":           "5807",
 	"pq.TestCopyInMultipleValues":        "5807",
 	"pq.TestCopyInRaiseStmtTrigger":      "5807",
+	"pq.TestCopyInStmtAffectedRows":      "5807",
 	"pq.TestCopyInTypes":                 "5807",
 	"pq.TestCopyInWrongType":             "5807",
 	"pq.TestCopyRespLoopConnectionError": "5807",
@@ -111,10 +113,6 @@ var libPQBlacklist19_2 = blacklist{
 var libPQIgnorelist20_1 = libPQIgnorelist19_2
 
 var libPQIgnorelist19_2 = blacklist{
-	// TestCloseBadConn fails due to an issue with the test.
-	// TODO(asubiotto): The upstream fix has been submitted so this can be removed
-	// from blacklists once we test with a newer version of lib/pq.
-	"pq.TestCloseBadConn": "41690",
 	// TestFormatTsBacked fails due to not returning an error for accepting a
 	// timestamp format that postgres does not.
 	"pq.TestFormatTsBackend": "41690",

